cargo-miri/src/util.rs

```diff
@@ -88,6 +88,77 @@ pub fn escape_for_toml(s: &str) -> String {
     format!("\"{s}\"")
 }
 
+pub fn flagsplit(flags: &str) -> Vec<String> {
+    // This code is taken from `RUSTFLAGS` handling in cargo.
+    // Taken from miri-script util.rs
+    flags.split(' ').map(str::trim).filter(|s| !s.is_empty()).map(str::to_string).collect()
+}
+
+pub fn get_miriflags_cargo_mini() -> Vec<String> {
```

Comment:
Please add a doc comment explaining what this does. (Seems like you have a bunch of that inside the function, but that's now how doc comments work in Rust.)

Why "mini"?